test/Common/Common.csproj

```diff
@@ -6,6 +6,8 @@
   </PropertyGroup>
 
     <ItemGroup>
+    <PackageReference Include="Microsoft.EntityFrameworkCore.InMemory" Version="5.0.15" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="5.0.0" />
```

Comment:
Should we create a new project for sharing integration code? Would remove the hard dependency on each csproj from Common.

Reply:
I think it's a good idea. I move it to it's own project.